### PR TITLE
Tunnistamo review instance

### PR DESCRIPTION
**Pull request by terovirtanen**
## Description :sparkles:
Review environment uses local tunnistamo (instead of KuVa tunnistamo-qa) and local kultus-api instances. 
Build arguments is possible set as pipeline parameter.

Most of changes has been done on kultus-pipelines [kultus-pipeline pr5830](https://dev.azure.com/City-of-Helsinki/kultus/_git/kultus-pipelines/pullrequest/5830)

### Closes :no_good_woman:
**[DEVOPS-547](https://helsinkisolutionoffice.atlassian.net/browse/DEVOPS-547)** 

### Manual testing :construction_worker_man:
Local kultus-api database is empty.  Admin-ui can login but there is not any other data.

## Additional notes :spiral_notepad:

[DEVOPS-547]: https://helsinkisolutionoffice.atlassian.net/browse/DEVOPS-547?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: azure-pipelines-review.yml [hunk 1]
```diff
@@ -46,5 +46,19 @@ extends:
...
+    # The values here will override the values defined in the harrastukset-pipelines repository.
+    buildArgs:
+      REACT_APP_API_URI: 'https://kultus-local.api.dev.hel.ninja/graphql'
+      REACT_APP_API_REPORT_URI: 'https://kultus-local.api.dev.hel.ninja/reports'
+
+      REACT_APP_LINKEDEVENTS_API_URI: 'https://api.hel.fi/linkedevents-test/v1'
+
+    # FIXME: The kultus.hkih -server responded several days with gateway timeout and internal server error, 
+    # so it was changed to the production cms.
+    # REACT_APP_CMS_URI: 'https://kultus.hkih.stage.geniem.io/graphql'
```
Comment (nikomakela) [inline azure-pipelines-review.yml:61]: Should work! If not, the CMS team should be contacted

File: azure-pipelines-review.yml [hunk 2]
```diff
@@ -46,5 +46,19 @@ extends:
...
\ No newline at end of file
+  parameters:
+    # Application build arguments and config map values as key value pairs.
+    # Does not contain all buildArguments, the rest located in kultus-pipelines/kultus-admin-ui-development.yml and kultus-admin-ui-common.yml
+    # The values here will override the values defined in the harrastukset-pipelines repository.
+    buildArgs:
+      REACT_APP_API_URI: 'https://kultus-local.api.dev.hel.ninja/graphql'
+      REACT_APP_API_REPORT_URI: 'https://kultus-local.api.dev.hel.ninja/reports'
+
+      REACT_APP_LINKEDEVENTS_API_URI: 'https://api.hel.fi/linkedevents-test/v1'
```
Comment (nikomakela) [inline azure-pipelines-review.yml:57]: Is this really available? At least it does not work with the production CMS! I think the new proper URL is `https://linkedevents.api.test.hel.ninja/v1/event/`

**Review by nikomakela — CHANGES_REQUESTED** [review]
The configurations does not fit together, so they do need some proposed changes.

**Comment by terovirtanen** [comment]
**kultus-admin-ui branch is deployed to platta: https://kultus-admin-ui-pr305.dev.hel.ninja** :rocket::rocket::rocket:

File: azure-pipelines-review.yml (hunk 1 above)
Comment (terovirtanen) [reply azure-pipelines-review.yml]: switch to kultus.hkih -server

File: azure-pipelines-review.yml (hunk 2 above)
Comment (terovirtanen) [reply azure-pipelines-review.yml]: done and this has changed to test pipeline too

**Comment by terovirtanen** [comment]
**kultus-admin-ui branch is deployed to platta: https://kultus-admin-ui-pr305.dev.hel.ninja** :rocket::rocket::rocket:

**Review by nikomakela — APPROVED** [review]

**Review by nikomakela — APPROVED** [review]

**Pull request merged**